b3lb/rest/b3lb/tasks.py

```diff
@@ -33,6 +34,33 @@
 #
 # Celery task routines
 #
+def housekeeping():
```

Comment:
This is referenced nowhere. I would prefer to use a more describing name like `cleanup_assets`. Maybe we could call this from `rest.tasks.check_slides` so that we recycle the old scheduled task?

Shouldn't we drop the `checkslides` manage command?